Sources/MockoloFramework/Templates/ClosureTemplate.swift

```diff
@@ -32,6 +32,9 @@ func applyClosureTemplate(name: String,
             if argType.typeName.hasPrefix(String.autoclosure) {
                 return argName + "()"
             }
+            if argName.isSwiftKeyword {
```

Comment:
remove this and use argName.safeName (see the comment below) everywhere in this block